Uplift/Views/HomeView.swift

```diff
@@ -71,18 +71,38 @@ struct HomeView: View {
             if let gyms = viewModel.gyms {
                 VStack(spacing: 12) {
                     HStack(spacing: 12) {
-                        capacityCircle(facility: gyms.facilityWithName(name: Constants.FacilityNames.hnhFitness))
+                        viewModel.capacityCircleNavLink(
```

Comment:
As mentioned above, the ViewModel should not contain anything View related. This function should be at the bottom of this file.